### PR TITLE
ENH Use SearchableMultiDropdownField

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-admin/issues/1618

CI requires https://github.com/silverstripe/silverstripe-framework/pull/11057 to be merged first

Also requires https://github.com/silverstripe/silverstripe-admin/pull/1635 to be merged

### CODE REVIEW

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
Don't forget to remove the `$membersMap` variable assignment

**Comment by emteknetnz** [comment]
Have removed $membersMap variable

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
LGTM, but needs a dependency bump for admin and framework.

**Comment by emteknetnz** [comment]
Have updated composer.json

**Review by GuySartorelli — APPROVED** [review]
LGTM, will merge when admin and framework PRs have been merged.

**Comment by GuySartorelli** [comment]
This is failing CI

**Comment by emteknetnz** [comment]
Have updated behat test, though will require this new PR to be merged first - https://github.com/silverstripe/silverstripe-admin/pull/1635

File: tests/behat/features/manage-page-permissions.feature [hunk 1]
```diff
@@ -90,9 +90,13 @@ Feature: Manage global page permissions
     And pages should not be editable by "SECURITY"
     But pages should be editable by "ADMIN"
 
+@sboyd
```
Comment (ChloeSartorelli) [inline tests/behat/features/manage-page-permissions.feature]:
```suggestion
```

File: tests/behat/features/manage-page-permissions.feature [hunk 2]
```diff
@@ -90,9 +90,13 @@ Feature: Manage global page permissions
...
   Scenario: I can limit global edit permissions to certain members
+    Given I take a screenshot after every step
```
Comment (ChloeSartorelli) [inline tests/behat/features/manage-page-permissions.feature]:
```suggestion
```

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
Passes locally but need to remove debug steps

File: tests/behat/features/manage-page-permissions.feature (hunk 1 above)
Comment (emteknetnz) [reply tests/behat/features/manage-page-permissions.feature]: Removed

File: tests/behat/features/manage-page-permissions.feature (hunk 2 above)
Comment (emteknetnz) [reply tests/behat/features/manage-page-permissions.feature]: Removed

**Review by GuySartorelli — APPROVED** [review]
LGTM

**Pull request merged**